app/helpers/remove-html-tags.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function removeHtmlTags(params) {
+  if (!(params[0] === undefined)) {
+    return jQuery("<p>" + params[0] + "</p>").text();
```

Comment:
The easiest (and most reliable) way to get access to jQuery is with `Ember.$` instead of using the global.